### PR TITLE
🏗️ upgrade monorepo structure for 3.x

### PR DESCRIPTION
**Pull request by ice-zjchen**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/qiankun/ISSUE_URL

### CODE REVIEW

File: docs/guide/tutorial.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The main app is not limited to the technical framework, it only needs to provide
 Install `qiankun` first :
 
 ```shell
-$ yarn add qiankun # or npm i qiankun -S
```
Comment (kuitos) [inline docs/guide/tutorial.md:16]: 用户不用使用 pnpm 装 qiankun 吧

File: docs/README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ footer: MIT Licensed | Copyright © 2019-present<br />Powered by [dumi](https://
 ## 📦 Installation
 
 ```shell
-$ yarn add qiankun  # or npm i qiankun -S
```
Comment (kuitos) [inline docs/README.md:22]: 用户侧不用改

**Review by kuitos — CHANGES_REQUESTED** [review]
examples 挪出去吧，我单独开一个 repo

**Comment by ice-zjchen** [comment]
> examples 挪出去吧，我单独开一个 repo

也无不可。好处是干净，坏处是不方便调试。我先删了

**Pull request merged**